src/components/DigitalCredentials/dialogs/ConfirmCredentialsTermsofUseDialog.vue

```diff
@@ -9,18 +9,7 @@
   >
```

Comment:
Have you considered a width of 60rem (line 5) so that the TOU is less tall?

Reply:
We could do that yeah. Just went with what the other dialogs are set at

Reply:
Up to you. The initial intent was dialog widths of 30rem and 45rem. Yours is special :)

Reply:
Looks good at 60rem wide!

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/049ee791-d411-4022-80b7-8e99263d6a4e)